### ISSUE TITLE
Diff code syntax highlighlighting

### ISSUE THREAD

**Issue by bfintal** [ahmadawais/shades-of-purple-vscode#29]
Awesome work as usual 🥇 

I would like to suggest adding diff syntax highlighting colors. For example, I have this in my markdown file:

```
```diff
Retained line
+ Added line
- Removed line
--- Comment
``
```

In Shades of Purple:
![screen shot 2019-01-10 at 10 39 51 pm](https://user-images.githubusercontent.com/1033611/50975503-c20cba80-1528-11e9-8cf7-2485c6d836ba.jpg)

In others:
![screen shot 2019-01-10 at 10 40 12 pm](https://user-images.githubusercontent.com/1033611/50975511-c638d800-1528-11e9-823b-aa2e40716599.jpg)

**Comment by ahmadawais** [comment]
FIXED in 46b5b465f386d62a0227dd56bb6ed771b8781987 and will be part of v5.0.0

**Comment by ahmadawais** [comment]
![diff](https://on.ahmda.ws/0a815b/c)
